src/analyses/flow_insensitive_analysis.cpp

```diff
@@ -46,19 +46,19 @@ exprt flow_insensitive_abstract_domain_baset::get_return_lhs(locationt to) const
   return to_code_function_call(to->code).lhs();
 }
 
-void flow_insensitive_analysis_baset::operator()(
-  const goto_functionst &goto_functions)
+void flow_insensitive_analysis_baset::
+operator()(const irep_idt &function_id, const goto_programt &goto_program)
 {
-  initialize(goto_functions);
-  fixedpoint(goto_functions);
+  initialize(goto_program);
+  goto_functionst goto_functions;
+  fixedpoint(function_id, goto_program, goto_functions);
```

Comment:
What are the goto_functions used for?

Reply:
They are used to resolve function calls in `do_function_call_rec` (though in this case it will just fail to resolve any function calls). That said, I have cleaned up the patch so that this change doesn't even show up anymore, because it was only an artefact of reordering functions in the file.